Networking/Poller.cc

```diff
@@ -154,6 +154,11 @@ namespace litecore { namespace net {
     }
 
 
+    void Poller::interrupt(int fd) {
+        Assert(fd > 0);
+        _interrupt(fd);
+    }
+
     void Poller::stop() {
         interrupt(-1);
```

Comment:
-1 here?

Reply:
Yes. The code on the background thread interprets -1 as a request to stop the thread.

Reply:
Do you mean _interrupt(), since the above assertion?

Reply:
Oh, I see -- yes, line 163 should be calling `_interrupt`. I'll fix it.